clients/index.js

```diff
@@ -417,3 +418,17 @@ ApplicationClients.prototype.updateKValues = function updateKValues() {
         self.serviceProxy.updateServiceChannels();
     }
 };
+
+ApplicationClients.prototype.updateKillSwitches = function updateKillSwitches() {
+    var self = this;
+    self.serviceProxy.unblockAll();
```

Comment:
Yes, this will.
Remote config will be the source of truth. So, the steps should be:
1. set up the remote config.
2. run admin command for immediate effect.